src/Symfony/Component/Validator/Constraints/CountryValidator.php

```diff
@@ -40,6 +41,10 @@ public function validate($value, Constraint $constraint)
             throw new UnexpectedTypeException($value, 'string');
         }
 
+        if (!class_exists(Intl::class)) {
+            throw new RuntimeException('The "symfony/intl" component is required to use the Country constraint.');
```

Comment:
hmm, shouldn't it be a `LogicException` instead?

Reply:
i think not, this is not a logic issue.

Reply:
looking at the code, we use LogicException quite often in similar situations, so could be worth for consistency at least (there are a few similar cases using RuntimeException, but they're not the majority and could be worth fixing)

Reply:
I would have used `RuntimeException` too ... but if we use `LogicException` in similar situations, it's better to keep consistency. @ronfroy can you please update this? Thanks!

Reply:
how can we use `LogicException` elsewhere if this PR introduces it as a new class?

For this case we use the `RuntimeException` already :+1: 
https://github.com/symfony/symfony/blob/645b01659c2d54e264c181df8d28ddea5f8b619a/src/Symfony/Component/Validator/Constraints/ExpressionValidator.php#L56-L63